resources/g2/sprites.json

```
@@ -962,6 +962,10 @@
         "forceBmp": true
     },
     {
+        "path": "font/latin/l-small.png",
+        "palette": "keep",
+        "forceBmp": true
+    },
         "path": "font/latin/c-circumflex-uc-small.png",
```

There is a syntax error here now due to a missing opening brace:
```suggestion
    {
        "path": "font/latin/c-circumflex-uc-small.png",
```

---

Note that the order in which these characters are defined matters for the sprite ids! In order for them to match the definitions in `sprites.h`, you should *append* them the end of the file, not put them in between.

---

Ahh you're right I missed that, I'll add those fixes.